renderer/components/ActivitiesSection.tsx

```diff
@@ -49,11 +48,13 @@ export default function ActivitiesSection({
       <div>
         <a
           href="#"
-          className="block px-4 py-4 text-sm font-medium text-center text-gray-500 bg-gray-50 hover:text-gray-700 sm:rounded-b-lg"
+          className="block px-4 py-4 text-sm font-medium text-center text-blue-600 bg-blue-200 hover:bg-blue-300  sm:rounded-b-lg"
           onClick={() => onEditActivity("new")}
```

Comment:
I would choose dark-gray color of button and white text. But I'll leave it up to you